src/parser/paladin/holy/constants.js

```diff
@@ -50,6 +51,7 @@ export const BEACON_TRANSFERING_ABILITIES = {
   [SPELLS.AVENGING_CRUSADER_HEAL_NORMAL.id]: 1,
   [SPELLS.AVENGING_CRUSADER_HEAL_CRIT.id]: 1,
   [SPELLS.GLIMMER_OF_LIGHT_HEAL_TALENT.id]: 0.25,
+  [SPELLS.WORD_OF_GLORY.id]: .5,
```

Comment:
This doesn’t make sense. Did you test it by selfhealing?

Reply:
Don't worry. There is a fix already coming. I pushed it then some of the holy people (before testing it) was like `yo this how it work`

Reply:
👍